src/views/Dashboard/GeneticAnalyst/Account/index.vue

```diff
@@ -553,7 +563,13 @@ export default {
       ],
       phoneNumber:  [
         rulesHandler.FIELD_REQUIRED,
-        val => /^[0-9]+$/.test(val) || "Phone number is invalid"
+        val => /^[0-9]+$/.test(val) || "Phone number is invalid",
+        rulesHandler.MAX_CHARACTER(12)
+      ],
+      profileImage: [
+        rulesHandler.FIELD_REQUIRED,
+        rulesHandler.FILE_SIZE(2000000),
+        (val) => (val && (val.type === "image/png" || val.type === "image/jpg")) || errorMessage.FILE_FORMAT("PNG/JPG")
```

Comment:
It would be better to make a `constant` array filled with the allowed image types and validate using `includes` rather than typing it out one by one.

Reply:
updated in this commit [bfd2701](https://github.com/debionetwork/debio-genetic-analyst-ui/pull/20/commits/bfd27014c83d684ae2dc0f7787c894e428b35878)